extensions/amp-list/amp-list.md

```diff
@@ -282,33 +282,62 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 If `binding` attribute is not provided, default is `always`.
 
 ## Experimental: Infinite Scroll (amp-list-load-more)
-We've introduced an experiment called `amp-list-load-more` as an implementation for pagination and infinite scroll in amp-list. This is an experimental feature, and final APIs may change.
+We've introduced the `amp-list-load-more` experiment as an implementation for pagination and infinite scroll in `amp-list`. This is an experimental feature, and final APIs may change.
 
 ### Attributes
 #### load-more (mandatory)
-Adding this attribute will allow amp-list (with no value) to show a “load-more" button at the end of the amp-list. The value of this attribute can be set to “auto" to trigger automatic loading more elements three viewports down for an infinite scroll effect.
+Adding this attribute allows amp-list (with no value) to show a “load-more" button at the end of the amp-list. The value of this attribute can be set to “auto" to trigger automatic loading more elements three viewports down for an infinite scroll effect.
```

Comment:
I was little confused that there was an option for a value and no value. Maybe something like:

"Adding this attribute with no value to amp-list will show a "load-more" button at the end of amp-list. To automatically load more elements, a value of "auto" can be added tot he attribute, amp-list will then automatically load elements three viewports down for an infinite scroll effect."

Can we also have a code sample for where this attribute is added? Unless it becomes obvious from the previous code sample.

Reply:
I'll need a code change for this, but there's an outstanding feature request to make this attribute take "auto" or "manual" and to be able to toggle this from an attribute in the JSON payload. It will probably be clearly if we require it to be `auto` or `manual` and invalidate the document if it is neither of these.

Reply:
I was also confused by this wording - an easy fix would be to move the parenthetical ("Adding this attribute with no value will allow amp-list... Setting its value to "auto" will...")

Longer term it seems like we'd want to come up with some kind of visual pattern to convey that this attribute can have a value or not and what each of those options does. There must be docs that do something like this (I'm imagining a mini-table) without making the reader parse several sentences of prose.

Reply:
I've made changes in the validator so that this now requires a value of `auto` or `manual`. I will update this shortly.